### PR TITLE
Adding more ecosystem sources

### PR DESCRIPTION
**Pull request by glasgowm148**

### CODE REVIEW

File: data/ecosystems/e/ergo-hodlcoin.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title = "HodlcoinERG"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/PhoenixErgo", ]
+
+# Repositories
+
+[[repo]]
+url = "https://github.com/pulsarz/hodlcoin-contracts"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-hodlcoin.toml:11]: can be removed as it doesn't exist

File: data/ecosystems/e/ergo.toml [hunk 2]
```diff
@@ -2,48 +2,79 @@
...
+
+    # Misc 
+    "ZenGate Global",
+    "Ergode",
+    "Ergo Names",
+    "LiliumErgo",
+    "Comet", 
+    "Ergo Stats and Analysis"
+
+    ]
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo.toml:77]: please keep the original alphabetical sorting as we have it as a standard in all the other toml files to make it easier to visually search for ecosystem names

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Overall, the PR looks great, just left a couple of small comments, please take a look, let's make the changes and then we can get it merged

**Comment by eherrerosj** [comment]
Some checks still not passing. Please lmk when you are ready for review again @glasgowm148

**Comment by eherrerosj** [comment]
Would you be able to wrap up this PR within the next couple weeks @glasgowm148?

**Comment by glasgowm148** [comment]
> Would you be able to wrap up this PR within the next couple weeks @glasgowm148?

Sorry, never got any notifications for this, will sort now

File: data/ecosystems/e/ergo.toml (hunk 2 above)
Comment (glasgowm148) [reply data/ecosystems/e/ergo.toml]: fixed

**Comment by eherrerosj** [comment]
> > Would you be able to wrap up this PR within the next couple weeks @glasgowm148?
> 
> Sorry, never got any notifications for this, will sort now

hey @glasgowm148, I discussed with the team internally and decided to actually add the missing tag to the repos that don't exist, instead of removing them as I initially proposed. Would you mind keeping the repos that you removed because they don't exist in the toml file and **optionally** add a missing tag when they return a 404 (`missing = true` below each repo link)? 

I say optionally because if you don't add it we handle it internally through our ETL process.

**Comment by glasgowm148** [comment]
> > > Would you be able to wrap up this PR within the next couple weeks @glasgowm148?
> > 
> > 
> > Sorry, never got any notifications for this, will sort now
> 
> hey @glasgowm148, I discussed with the team internally and decided to actually add the missing tag to the repos that don't exist, instead of removing them as I initially proposed. Would you mind keeping the repos that you removed because they don't exist in the toml file and **optionally** add a missing tag when they return a 404 (`missing = true` below each repo link)?
> 
> I say optionally because if you don't add it we handle it internally through our ETL process.

The one you suggested isn't missing though, just a fork that was deleted. Original repo is still present.

What is the missing tag?

**Comment by eherrerosj** [comment]
> What is the missing tag?

@glasgowm148, an example of how to add the missing tag [here](https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/r/rankradaricp.toml#L11)

**Comment by eherrerosj** [comment]
Hi @glasgowm148, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.

Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

**Comment by eherrerosj** [comment]
Closing due to inactivity, please reopen when ready

**Comment by glasgowm148** [comment]
> Hi @glasgowm148, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.
> 
> Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

Why exclude months worth of links due to some policy change on missing URLs? What is missing, some fork that was deleted?

Please merge as is or fix on your end.

**Pull request closed without merging**